.github/workflows/integration_test.yaml

```diff
@@ -8,7 +8,7 @@ jobs:
     uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
     secrets: inherit
     with:
-      extra-arguments: --localstack-address 172.17.0.1
+      extra-arguments: --localstack-address 172.17.0.1 -m "not cos"
```

Comment:
Would it make more sense to remove these tests? If they're not run, they'll probably stop working after a while. As they are checked in version control, they can be restored later if you wish.

Reply:
We are planning to run them before publishing to edge :)

Reply:
Gotcha, could you please add this to the PR description?

Reply:
Sure, thanks for raising this.